lambda/script.py

```diff
@@ -0,0 +1,154 @@
+import json
+import os
+from typing import Dict, List, Optional
+
+import boto3  # type: ignore
+import googleapiclient.discovery  # type: ignore
+from google.oauth2 import service_account  # type: ignore
+
+
+def get_env_var(env) -> Optional[str]:
+    """
+    Returns an enviroment variable.
+    """
+    return os.environ.get(env)
+
+
+def get_scope(scope: Optional[str]) -> List[str]:
+    """
+    Returns the scope from AWS SSM.
+    """
+    client = boto3.client("ssm")
+    return [client.get_parameter(Name=scope, WithDecryption=True)["Parameter"]["Value"]]
+
+
+def get_subject_email(subject: Optional[str]) -> str:
+    """
+    Returns the subject email from AWS SSM
+    """
+    client = boto3.client("ssm")
```

Comment:
I think it's best practice to create the client once at the top-level of the file?

Reply:
This has been done.